core/templates/pages/landing-pages/stewards-landing-page/stewards-landing-page.controller.spec.ts

```diff
@@ -69,7 +69,7 @@ describe('Stewards Landing Page', function() {
       href: '/library'
     }, {
       text: 'Subscribe to our Newsletter',
-      href: 'https://tinyletter.com/oppia'
+      href: 'http://eepurl.com/g5v9Df'
```

Comment:
Just one note -- here and elsewhere, could you please use https instead of http?

Reply:
Done!